charts/ovirt-csi-driver-4.21.0-alpha1/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: ovirt-csi-driver
+description: Implementation of a CSI driver for oVirt.
+type: application
+version: 4.21.0-alpha1
+appVersion: "4.21.0-alpha1"
+icon: icons/KubeVirt_icon.png
```

Comment:
Is there a reason the kubevirt icon is being used?

Reply:
not sure why we picked that, @mgianatagh do you know?  It is being used for 4.20.0 also.